packages/react-ui-core/src/Ratings/__tests__/__snapshots__/RatingBar-test.js.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`RatingBar ratings fills 1 of 5 rating items 1`] = `"<div class=\\"\\"><div class=\\"\\"><div class=\\"\\" style=\\"width:20%\\"></div></div></div>"`;
```

Comment:
These snapshots are not getting the expected classnames, perhaps need to mock a theme to be passed into the component so the classnames get added.

Need to add coverage for a developer specifying a different classname either using className or RatingIcon props.